sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/DoNotMarkEnumsWithFlags.cs

```diff
@@ -81,6 +81,7 @@ protected override void Initialize(SonarAnalysisContext context)
 
         private static ulong? GetEnumValueOrDefault(IFieldSymbol enumMember)
```

Comment:
Sorry, should have spotted that before. Will the test pass these test cases? Especially the first one - wouldn't it crash at otherValues[0]?

```
    enum EnumFoo { }

    enum EnumFoo2
    {
        a = 1,
        b = 2,
        c = 15
    }

    enum EnumFoo3
    {
        a = 2,
        b = 4,
        c = 4,
        d = 10
    }
```